AppActionVisualizer/app/src/main/java/com/example/appactionvisualizer/utils/Utils.java

```diff
@@ -22,11 +25,11 @@
 import java.util.List;
 
 public class Utils {
```

Comment:
Rename this class to differentiate from StringUtils?

Reply:
sure, rename to appUtils